Doc/library/sysconfig.rst

```diff
@@ -430,8 +430,8 @@ Other functions
 
 .. _sysconfig-cli:
 
-Using :mod:`sysconfig` as a script
```

Comment:
Let's also add the `#using-sysconfig-as-a-script` anchor, which is what we get from clicking the header in the ToC.